modules/library/render/src/main/java/org/geotools/renderer/style/MarkFactoryListComparator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.renderer.style;
+
+import java.util.Comparator;
+import java.util.List;
+
+/** Mark factory comparator based on a simple class name identifiers list. */
+public class MarkFactoryListComparator implements Comparator<MarkFactory> {
```

Comment:
Does it need to be public?

Reply:
Yes, this class is used on GeoServer rendering package to inject the Comparator hint.